src/main/java/org/spongepowered/api/command/CommandMessageFormatting.java

```diff
@@ -25,17 +25,16 @@
 package org.spongepowered.api.command;
 
 import org.spongepowered.api.text.Text;
-import org.spongepowered.api.text.Texts;
 import org.spongepowered.api.text.format.TextColors;
 
 public class CommandMessageFormatting {
-    public static final Text PIPE_TEXT = Texts.of("|");
-    public static final Text SPACE_TEXT = Texts.of(" ");
-    public static final Text PLUS_TEXT = Texts.of("*");
-    public static final Text NEWLINE_TEXT = Texts.of('\n');
-    public static final Text LT_TEXT = Texts.of("<");
-    public static final Text GT_TEXT = Texts.of(">");
-    public static final Text ELIPSES_TEXT = Texts.of("…");
+    public static final Text PIPE_TEXT = Text.of("|");
+    public static final Text SPACE_TEXT = Text.of(" ");
+    public static final Text PLUS_TEXT = Text.of("*");
+    public static final Text NEWLINE_TEXT = Text.of('\n');
```

Comment:
The char version is supported now as well. [`Text.of(char)`](https://github.com/SpongePowered/SpongeAPI/pull/988/files#diff-c94a2b7359796a94a693e89c08e6dfacR689)